### PR TITLE
Adds some stuff to charlie station

### PR DESCRIPTION
**Pull request by redmoogle**
# Document the changes in your pull request

Adds a thing of cable coils by medical maint
stuffs the PA in a crate
Removes the transport pod for a bridge
Makes the science room *slighty* bigger
Makes the security room slightly smaller
Adds a small randomized maint loot
gets rid of unbreakable rocks
fixes two vents being off
removes random unconnected wire

image

## Why?

Well; The issue is that the pod gets in the way alot and so I changed it to a foot bridge which caused me to need to realign some stuff which is why theres now two maint rooms

I also included three fixes because why not

# Wiki Documentation

Just remove the part saying they're connected by transit tubes

# Changelog

:cl:  
rscadd: Removes the pod from charlie and adds a bridge instead
tweak: Stuff the PA in charlie station into a crate
tweak: Adds another pack of cable coils to charlie station
bugfix: Fixes some issues on charlie station
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Why?
Also include the wiki part

**Comment by redmoogle** [comment]
> 
> 
> Why? Also include the wiki part

added one stack of cable coils because you kinda run out
added a bridge because pod is annoying and weird
PA just takes up alot of unneeded room
The rest is to make the alignment work

**Comment by Moggram** [comment]
The AI core is now open to space.
You need to add some stuff in the new maintenance rooms so they are not empty.

**Comment by Chubbygummibear** [comment]
Something important to note, I'm pretty sure you bulldozed through one of the 4 bluespace crystal spawns with the footbridge. I recently learned there are 4 distinct spawns, but you need 5 to make a functional teleporter. 

So if I may make a suggestion, re-add the bluespace crystal spawn, and maybe add a few diamonds to one of the bridge crates so it's feasible to make artificial bluespace crystals without hoping for a lucky diamond ore spawn for the 5th crystal. 

That aside, love to see ghost role love bless up

**Comment by redmoogle** [comment]
> 
> 
> Something important to note, I'm pretty sure you bulldozed through one of the 4 bluespace crystal spawns with the footbridge. I recently learned there are 4 distinct spawns, but you need 5 to make a functional teleporter.
> 
> So if I may make a suggestion, re-add the bluespace crystal spawn, and maybe add a few diamonds to one of the bridge crates so it's feasible to make artificial bluespace crystals without hoping for a lucky diamond ore spawn for the 5th crystal.
> 
> That aside, love to see ghost role love bless up

Readded that one crystal and also put down an diamond asteroid tile

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 1]
```diff
@@ -7089,6 +6982,9 @@
 /obj/machinery/door/firedoor/border_only{
 	dir = 8
 	},
+/obj/machinery/door/airlock/maintenance_hatch{
+	name = "Electrical Maintanace"
```
Comment (patpol4) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: maintanace

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 2]
```diff
@@ -7393,6 +7252,10 @@
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
 	dir = 4
 	},
+/obj/machinery/door/airlock/maintenance_hatch{
+	name = "Electrical Maintanace"
```
Comment (patpol4) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: maintanace

**Review by patpol4 — CHANGES_REQUESTED** [review]
1. Plant is fine but what is the cobweb hanging onto?
image

2. I don't get the point of this change
image

Other than this and the maintanace stuff, this is very nice.

**Comment by redmoogle** [comment]
> 1. Plant is fine but what is the cobweb hanging onto?
> image
> 
> 2. I don't get the point of this change
> image
> 
> Other than this and the maintanace stuff, this is very nice.

1 oop 

2 those were indestructible turfs and changed it to look like an actual maint/storage room

**Comment by patpol4** [comment]
just the maint typos now and its good to go

**Comment by redmoogle** [comment]
> 
> 
> just the maint typos now and its good to go

fixed

**Review by patpol4 — APPROVED** [review]
Is good

**Pull request merged**